src/node/history.h

```diff
@@ -208,20 +208,21 @@ namespace ccf
   private:
     uint64_t index;
     uint32_t max_index;
+    static const uint32_t hash_size = 32U;
```

Comment:
I don't think we need that, that's just Sha256Hash::SIZE.

Reply:
That's right, done.